### ISSUE TITLE
Weird character encoding for `:` character in URLs

### ISSUE THREAD

**Issue by napulen** [DDMAL/cantus#537]
Linking the manifest 

https://adore.ugent.be/IIIF/manifests/archive.ugent.be:082FD364-C35A-11DF-A9D6-99EF78F64438 

to Manuscript `Utrecht, NL-Uu 406 (3 J 7)` results in `Diva.js` not being able to construct URLs that fetch any images. 

Like this one:

https://adore.ugent.be/IIIF/images/archive.ugent.be%253A082FD364-C35A-11DF-A9D6-99EF78F64438%253ADS.2/0,0,2048,2048/256,256/0/default.jpg

which always return a `404` response from the server.

Looking more closely into the URLs in the manifest, it seems the generated URLs are okay, except for the encoding of the `:` character, which seems to break things.

In short:

- This works: https://adore.ugent.be/IIIF/images/archive.ugent.be:082FD364-C35A-11DF-A9D6-99EF78F64438:DS.18/0,0,2048,2048/256,256/0/default.jpg
- This doesn't: https://adore.ugent.be/IIIF/images/archive.ugent.be%253A082FD364-C35A-11DF-A9D6-99EF78F64438%253ADS.2/0,0,2048,2048/256,256/0/default.jpg

The second one was encoded by `Diva.js` (?)  and it seems the encoding of the colon is not appropriate.

Maybe a `utf-8`/`utf-16` thing?
